.github/release.yml

```
@@ -0,0 +1,19 @@
+changelog:
+  exclude:
+    labels:
+      - ignore-for-release
+    authors:
+      - JohnnyQ5
```

See newest template release 
```suggestion
      - JohnnyQ5
      - github-actions
```